appleseed-Installer/Product.wxs

```diff
@@ -0,0 +1,632 @@
+          </Component>
+          <Component Id='appleseed_max2019_impl.dll' Guid='{59436677-09A2-40E9-BEA8-3946C8C37D42}' Permanent='no' SharedDllRefCount='no' Transitive='no'>
+            <File Id='_appleseed_max2019_impl.dll' DiskId='1' Hidden='no' ReadOnly='no' System='no' Vital='yes'  Name='appleseed-max2019-impl.dll' Source='..\appleseed-max2019\appleseed-max2019-impl.dll' KeyPath='yes' />
+          </Component>
+          <Component Id='appleseed_max2020.dlr' Guid='{22807B36-11FF-476A-982E-D215C947DD15}' Permanent='no' SharedDllRefCount='no' Transitive='no'>
+            <File Id='_appleseed_max2020.dlr' DiskId='1' Hidden='no' ReadOnly='no' System='no' Vital='yes'  Name='appleseed-max2020.dlr' Source='..\appleseed-max2020\appleseed-max2020.dlr' KeyPath='yes' />
+          </Component>
+          <Component Id='appleseed_max2020_impl.dll' Guid='{AC85B8A9-9B7A-457C-9A40-96BF6D1B3952}' Permanent='no' SharedDllRefCount='no' Transitive='no'>
+            <File Id='_appleseed_max2020_impl.dll' DiskId='1' Hidden='no' ReadOnly='no' System='no' Vital='yes'  Name='appleseed-max2020-impl.dll' Source='..\appleseed-max2020\appleseed-max2020-impl.dll' KeyPath='yes' />
+          </Component>
+          <Component Id='appleseed_max2021.dlr' Guid='{AA580367-B1B7-46F4-ACEB-CEE5E2A738CC}' Permanent='no' SharedDllRefCount='no' Transitive='no'>
+            <File Id='_appleseed_max2021.dlr' DiskId='1' Hidden='no' ReadOnly='no' System='no' Vital='yes'  Name='appleseed-max2021.dlr' Source='..\appleseed-max2021\appleseed-max2021.dlr' KeyPath='yes' />
+          </Component>
+          <Component Id='appleseed_max2021_impl.dll' Guid='{D372374A-09F0-48FD-B752-FB6190C4C7E0}' Permanent='no' SharedDllRefCount='no' Transitive='no'>
+            <File Id='_appleseed_max2021_impl.dll' DiskId='1' Hidden='no' ReadOnly='no' System='no' Vital='yes'  Name='appleseed-max2021-impl.dll' Source='..\appleseed-max2021\appleseed-max2021-impl.dll' KeyPath='yes' />
+          </Component>
+          <Component Id='appleseed.dll' Guid='{6DE34BD9-582d-4A63-8C1E-923608DE4B11}' Permanent='no' SharedDllRefCount='no' Transitive='no'>
+            <File Id='_appleseed.dll' DiskId='1' Hidden='no' ReadOnly='no' System='no' Vital='yes'  Name='appleseed.dll' Source='..\appleseed-max2017\appleseed.dll' KeyPath='yes' />
+          </Component>
+
+          <Directory Id='SHADERS' Name='shaders'>
+            <Component Id='dirSHADERS' Guid='{6990F077-1399-4D64-BF2C-15288342CF7B}' Transitive='no'>
+              <CreateFolder Directory='SHADERS' />
+            </Component>
+            <Directory Id='APPLESEED' Name='appleseed'>
+              <Component Id='dirAPPLESEED' Guid='{7A579FBA-240C-4103-8134-8987FA51C1B3}' Transitive='no'>
+                <CreateFolder Directory='APPLESEED' />
+              </Component>
+              <Component Id="as_anisotropy_vector_field.oso" Guid="{D32C03B8-CD7A-4396-B1B3-17EB860F0710}" Permanent='no' SharedDllRefCount='no' Transitive='no'>
```

Comment:
Is there a way to instead reference a kind of "list file" that contains the list of files we would like to install?

It's bound to happen that we will add/remove/rename shader files but completely forget to update this file...